packages/plugin-solid/src/rename-import.ts

```diff
@@ -0,0 +1,19 @@
+import { createVirtualModule } from '@rsbuild/shared';
+import { readFileSync } from 'fs';
+import type { PluginObj } from '@babel/core';
+
+export default function renameImport(): PluginObj {
+  return {
+    visitor: {
+      ImportDeclaration(path) {
+        if (path.node.source.value === 'solid-refresh') {
+          const runtimeFilePath = require.resolve(
+            'solid-refresh/dist/solid-refresh.mjs',
```

Comment:
Can we use alias to redirect the import path? The performance of alias is better than Babel plugin.

Reply:
I'll give it a try~